crates/services/p2p/src/behavior.rs

```diff
@@ -98,21 +115,42 @@ impl<Codec: NetworkCodec> FuelBehaviour<Codec> {
 
         let peer_report = PeerReportBehaviour::new(p2p_config);
 
+        let identify = {
+            let identify_config = identify::Config::new(
+                "/fuel/1.0".to_string(),
```

Comment:
Would we want to make a todo to make this configurable?
i.e something like
```suggestion
                format!("/fuel/{chain_id}/{p2p_protocol_version}", chain_id, P2P_VERSION),
```

Reply:
https://github.com/FuelLabs/fuel-core/issues/1564